vuepress/docs/next/tutorials/create/ms/generate-microservices-and-micro-frontends.md

```diff
@@ -1,81 +1,67 @@
 
-The general flow of component generation is:
-
-1. Run the Entando Blueprint to create your components (Spring Boot microservice and optionally React micro frontends)
-2. Customize and enhance your generated code
-3. Build an Entando Bundle from your components
-4. Deploy a [custom resource](../../../docs/consume/custom-resources.md) for your bundle into Kubernetes
-5. Install your Entando Bundle into your Entando Application(s)
+The output of this tutorial is [a new bundle project](../../../docs/curate/bundle-details.md#bundle-development-process) with several components: 
+- a Spring Boot microservice with CRUD operations for a single database entity
+- three React micro frontends for displaying and managing the entity
 
 ## Prerequisites
-Use the [Entando CLI](../../../docs/getting-started/entando-cli.md#check-the-environment) to verify environmental dependencies (e.g. Java, npm, git, JHipster, Entando Blueprint).
-``` sh
-ent check-env develop
-```  
-
-## Generate the Project
-Create a project with microservices.
-
-1. Setup a new project directory
-``` sh
-mkdir testProject && cd testProject
+- [A working instance of Entando](../../../docs/getting-started/)
+- Verify dependencies with the [Entando CLI](../../../docs/getting-started/entando-cli.md#check-the-environment): `ent check-env develop`
+
+## Set Up a New Bundle Project
+1. Create a new bundle project directory. This will add a simple `entando.json` descriptor as a starting point.
+```shell
+ent bundle init YOUR-PROJECT-NAME
+````
+
+## Generate the Components
+1. From the project directory, use the Entando CLI to add a basic microservice configuration to the project:
+```shell
+cd YOUR-PROJECT-NAME
+ent bundle ms add conference-ms --stack=spring-boot
 ```
 
-2. Use `ent jhipster` to generate the project skeleton via the Entando Blueprint
-``` sh
-ent jhipster --blueprints entando
+2. From the `conference-ms` directory, use the Entando Blueprint (powered by JHipster) to generate the `conference-ms` microservice:
+```shell
+cd microservices/conference-ms
+ent jhipster --blueprints=entando
 ```
 
-3. This triggers a project initialization prompt. Enter "Yes" in response.
-
-`The project dir doesn't seem to be initialized, should I do it now?` (**Yes**)
-
-4. Enter "Yes" when prompted with the following overwrite to resolve the conflict:
-
-`Overwrite .gitignore?`
-
-5. You'll be presented with a series of additional prompts pertaining to project configuration. These are echoed below, with the base values for this tutorial in parentheses. Input your preferences, except where a required entry is identified in **bold**. Note that the `Enter` key will select the default option.
-
+3. You'll be presented with a series of prompts pertaining to service generation. These are echoed below, with the base values for this tutorial in parentheses. Input your preferences, except where a required entry is identified in **bold**. Note that the `Enter` key will select the default option.
```

Comment:
my questions/selections didn't match this exactly; i can show you what printed to the terminal

Reply:
@Lyd1aCla1r3 the questions can differ based on answers to some questions. I went through this with the latest blueprint and thought it was consistent but let's look at what you're seeing.